telluric/georaster.py

```diff
@@ -2061,10 +2068,34 @@ def colorize(self, colormap, band_name=None, vmin=None, vmax=None):
 
         return self.copy_with(image=array, band_names=['red', 'green', 'blue'])
 
-    def _as_in_memory_geotiff(self):
-        temp_path = "/vsimem/%s.tif" % (uuid.uuid4())
-        self.save(temp_path)
-        return GeoRaster2.open(temp_path)
+    def _as_in_memory_geotiff(self, tags=None, extension="tif", **kwargs):
```

Comment:
Just for my curiosity, why adding tags, extension and **kwargs?

Reply:
Cause I see then you add 
``` internal_mask = kwargs.get('GDAL_TIFF_INTERNAL_MASK', True)
        nodata_value = kwargs.get('nodata', self.nodata_value)
        compression = kwargs.get('compression', Compression.lzw)
        rasterio_envs = {'GDAL_TIFF_INTERNAL_MASK': internal_mask}

Reply:
that original .save call was not considering (If I understood correctly)

Reply:
I copied most of the contents of this method from original `save`, including the optional parameters and these 3 lines. The original `_as_in_memory_geotiff` used to call `save` using the default params so we already know the values that will be used for this 3 variables and we could actually hardcode them.
As I don't fully understand how all options contribute at the exporting task I left them in case some tuning is needed at some point knowing that this is a private method and changing its signature is not a problem maybe?